chain-config/src/validator_rules.rs

```diff
@@ -15,43 +17,21 @@ pub struct TokenIdAmountPair<M: ManagedTypeApi> {
 pub trait ValidatorRulesModule {
     fn require_config_set(&self) {
```

Comment:
```suggestion
    fn require_valid_config(&self) {
```
Also, we can send a config as argument and validate it. This should be better because we first check the config, then we set it in storage